CHANGELOG.md

```diff
@@ -1,250 +1,310 @@
+| Fixed a bug that overwrote the animation-fill-mode                                          |  **Bugfix**  | [#1839][] |
+| Fixed talon import in pagination                                                            |  **Bugfix**  | [#1826][] |
+| Fixed SearchBar lazy loading                                                                |  **Bugfix**  | [#1764][] |
+| Removed back arrow for first level in main menu                                             |  **Bugfix**  | [#1337][] |
+| Fixed a transparent background bug for `.png` images                                        |  **Bugfix**  | [#1690][] |
 
 ### Peregrine library
 
-| Description                                                                | Change type  |    PR     |
-| :------------------------------------------------------------------------- | :----------: | :-------: |
-| Added a window size hook                                                   | **Feature**  | [#1193][] |
-| Added Toast feature                                                        | **Feature**  | [#1218][] |
-| Added eslint plugin for react hooks                                        | **Feature**  | [#1358][] |
-| Added a `withLogger` util function for `useReducer`                        | **Feature**  | [#1374][] |
-| Added a REST API Hook                                                      | **Feature**  | [#1321][] |
-| Added a Peregrine context provider                                         | **Feature**  | [#1316][] |
-| Added support for List initial selections                                  | **Feature**  | [#1589][] |
-| Removed Storybook from peregrine and update references                     |  **Update**  | [#1482][] |
-| Changed prop type of `uniqueID` for list Item component                    |  **Update**  | [#1586][] |
-| Refactored the image carousel component into hooks and visual components   | **Refactor** | [#1268][] |
-| Refactored list components to use hooks                                    | **Refactor** | [#1515][] |
-| Fixed a bug that caused a pagination double fetch                          |  **Bugfix**  | [#1381][] |
-| Fixed peregrine watcher and watch:all                                      |  **Bugfix**  | [#1378][] |
-| Fixed ApolloContext hook                                                   |  **Bugfix**  | [#1467][] |
-| Fixed bug in MagentoRouteHandler so it can correctly handle unknown routes |  **Bugfix**  | [#1495][] |
+| Description                                                                                   | Change type  |    PR     |
+| :-------------------------------------------------------------------------------------------- | :----------: | :-------: |
+| Implemented Product Images pre-fetching                                                       | **Feature**  | [#1938][] |
+| Created a talon for the Search root component                                                 | **Feature**  | [#1953][] |
```

Comment:
Should this be consistent with the other talons here? For example, should this instead read: "Created `<name of search root component>` talon"